### ISSUE TITLE
Show total size/delta in comment

### ISSUE THREAD

**Issue by kuldeepkeshwar** [kuldeepkeshwar/size-plugin-bot#21]
Currently, we are showing comment in PR like 👇
----
size-plugin-browser:
```
   main.js ⏤  1.11 kB (-24.8 kB ✅)
 1.main.js ⏤  207 B (-2 B)

```
----

I would like to add overall size/delta to the comment something like  👇
----
size-plugin-browser: Overall Size 1.11 kB (-24.8 kB ✅)
```
   main.js ⏤  1.11 kB (-24.8 kB ✅)
 1.main.js ⏤  207 B (-2 B)

```
----

**Comment by kuldeepkeshwar** [comment]
fixed in  #22

**Comment by kuldeepkeshwar** [comment]
fixed in #22
